app/Http/Controllers/Store/CentreController.php

```diff
@@ -201,6 +205,10 @@ private function getCentreRegistrationsSummary(array $centre_ids, $dateFormats =
             // Would be confusing if an old reason was left in - so check leaving date is there.
             $row["Leaving Reason"] = $reg->family->leaving_on ? $reg->family->leaving_reason : null;
 
+            if ($user->role === "foodmatters_user") {
```

Comment:
This check may not be necessary, if we pass "Date file was downloaded" in $excludeColumns